src/audio/dai-zephyr.c

```diff
@@ -797,6 +797,7 @@ static int dai_set_dma_buffer(struct dai_data *dd, struct comp_dev *dev,
 	uint32_t period_count;
 	uint32_t period_bytes;
 	uint32_t buffer_size;
+	uint32_t buffer_size_preferred;
```

Comment:
thinking aloud - do we need the extra member i.e. could we just re-write ```buffer_size``` and emit a log that says "cant create buffer size X, using size Y instead"

Reply:
@lgirdwood I'm sorry, but I don't understand. This is not a new structure field, just a local variable. The allocation function takes two sizes, the minimum one (`perdiod bytes * period bytes`) and the preferred one coming from a ipc.

Reply:
Got you - it looked like a struct when I reviewed. Good for me.